src/WebJobs.Script.WebHost/Diagnostics/DiagnosticEventTableStorageRepository.cs

```diff
@@ -55,18 +55,17 @@ public class DiagnosticEventTableStorageRepository : IDiagnosticEventRepository,
             ILogger<DiagnosticEventTableStorageRepository> logger)
             : this(configuration, hostIdProvider, environment, scriptHost, logger, LogFlushInterval) { }
 
-        internal CloudTableClient TableClient
+        internal TableServiceClient TableClient
         {
             get
             {
                 if (!_environment.IsPlaceholderModeEnabled() && _tableClient == null)
                 {
                     string storageConnectionString = _configuration.GetWebJobsConnectionString(ConnectionStringNames.Storage);
                     if (!string.IsNullOrEmpty(storageConnectionString)
-                        && CloudStorageAccount.TryParse(storageConnectionString, out CloudStorageAccount account))
+                        && Microsoft.Azure.Storage.CloudStorageAccount.TryParse(storageConnectionString, out _))
```

Comment:
I think you can switch to a `try/catch`, remove `Microsoft.Azure.Storage.CloudStorageAccount.TryParse`, and add the log statement in the `catch` block